eventually/manage.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+# pylint: disable=missing-docstring
```

Comment:
unless Is this file not ignored in the file _.pylintrc_?
`+ignore=CVS, migrations, apps.py, urls.py, wsgi.py, settings.py, local_settings.py, manage.py`

Reply:
despite "manage.py" is have to be ignored, the command `pylint --rcfile=.pylintrc *` verify it and returns that two errors that I've explicitly commented.

Reply:
ok, can you merge.